extensions/smallrye-reactive-messaging-amqp/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/amqp/deployment/AmqpDevServicesProcessor.java

```diff
@@ -277,14 +278,15 @@ private ArtemisContainer(DockerImageName dockerImageName, String extra, int fixe
             super(dockerImageName);
             this.port = fixedExposedPort;
             withNetwork(Network.SHARED);
-            withExposedPorts(AMQP_PORT);
+            withExposedPorts(AMQP_PORT, AMQP_CONSOLE_PORT);
             withEnv("AMQ_USER", DEFAULT_USER);
             withEnv("AMQ_PASSWORD", DEFAULT_PASSWORD);
             withEnv("AMQ_EXTRA_ARGS", extra);
             if (serviceName != null) { // Only adds the label in dev mode.
                 withLabel(DEV_SERVICE_LABEL, serviceName);
             }
-            if (dockerImageName.getRepository().endsWith("artemiscloud/activemq-artemis-broker")) {
+            if (dockerImageName.getRepository().startsWith("artemiscloud/activemq-artemis-broker")) {
```

Comment:
Why this change?

Reply:
Good point. This remains when we try to use other broker images based on artemiscloud. I will remove it.